Common/Securities/SecurityTransactionManager.cs

```diff
@@ -190,7 +190,7 @@ public List<OrderTicket> CancelOpenOrders(Symbol symbol, string tag = null)
             }
 
             var cancelledOrders = new List<OrderTicket>();
-            foreach (var ticket in GetOrderTickets(x => x.Symbol == symbol && x.Status.IsOpen()))
+            foreach (var ticket in GetOpenOrderTickets(x => x.Symbol == symbol))
```

Comment:
Did we check for all usages of `GetOrderTickets` and `GetOrders` to make a similar change?